command/connect/envoy/bootstrap_config.go

```diff
@@ -305,6 +307,18 @@ func (c *BootstrapConfig) generateStatsSinkJSON(name string, addr string) (strin
 	}`, nil
 }
 
+func (c *BootstrapConfig) statsSinkEnvMapping(s string) string {
+	allowedStatsSinkEnvVars := map[string]bool{
+		"HOST_IP": true,
+	}
+
+	if !allowedStatsSinkEnvVars[s] {
+		// if the specified env var isn't explicitly allowed, unexpand it
+		return fmt.Sprintf("${%s}", s)
```

Comment:
Could also just return an empty string. no strong preference here; either way, if someone enters a disallowed env var, it's going to fail `url.Parse`, so the result here is just cosmetic.